test/check-application

```diff
@@ -1970,10 +1970,12 @@ class TestFiles(testlib.MachineCase):
         m = self.machine
 
         self.enter_files()
+        b.click("button[aria-label='Display as a list']")
 
         # Copy/paste file
         m.execute("runuser -u admin mkdir /home/admin/newdir")
         m.write('/home/admin/newfile', 'test_text\n', owner='admin:admin')
+        b.wait_in_text("[data-item='newfile'] .item-owner", "admin")
```

Comment:
This feels like an independent change? Fixing a test flake or missing coverage or so? Please move into a separate commit (unless I'm missing something)